docs/v1/P4Runtime-Spec.mdk

```diff
@@ -4577,6 +4613,8 @@ semantics.
   metadata in the data plane set to the deleted `multicast_group_id` will be
   dropped.
 
+The support for back replicas is optional by the target and if the target does
```

Comment:
Please move this under INSERT, so we have the full specification will all error codes in one place.